mesa/space.py

```diff
@@ -632,6 +632,11 @@ def iter_neighborhood(
             equals at most 9 (8) if Moore, 5 (4) if Von Neumann (if not
             including the center).
         """
+        cache_key = (pos, include_center, radius)
+        neighborhood = self._neighborhood_cache.get(cache_key, None)
+
+        if neighborhood is not None:
+            return neighborhood
```

Comment:
Shouldn't this be `yield from neighborhood`?

Reply:
ops, sure